3-extra/2-array-of-objects.js

```diff
@@ -11,7 +11,11 @@
 */
 
 function getHighestRatedInEachGenre(books) {
-    // TODO
+    for(let everyObjects of books){
```

Comment:
I think you know this function isn't complete 😂 

If you need help with this one, please talk in the hawk channel, myself and the others can help you out.